SECURITY.md

```diff
@@ -0,0 +1,26 @@
+# Security Policy
+
+## Reporting a Security Issue
+
+To report a security issue, please email ??? with a description of the issue,
```

Comment:
In general, I think this is important to have. If someone finds a security concern in any of our projects, where do they email?

Reply:
scala.center@epfl.ch